test/libsolidity/semanticTests/various/value_insane.sol

```diff
@@ -21,7 +21,7 @@ contract test {
 // compileViaYul: also
 // ----
 // constructor(), 20 wei ->
-// gas irOptimized: 187835
+// gas irOptimized: 195125
```

Comment:
This is very interesting - why is there such a big code size change?

Reply:
It seems the reason is that it does not inline a particular function anymore (because the function to inline into gets a little larger)

Reply:
There are quite a few cases with high increases like this. Much more than I'd expect from a change that basically adds a single `if` per external call.

Do you think adjusting inlining heuristics will be enough to deal with it?